helm/polaris/templates/configmap.yaml

```diff
@@ -6,9 +6,9 @@
   to you under the Apache License, Version 2.0 (the
   "License"); you may not use this file except in compliance
   with the License.  You may obtain a copy of the License at
- 
+
```

Comment:
Not a big deal, but you should strive not to introduce such changes since it's not related to the PR.